tests/fixtures/simple/openapi.yaml

```diff
@@ -1021,10 +1039,10 @@ components:
       required:
         - image_version
   parameters:
-    OptionalName:
+    Name:
       name: name
       in: path
       description: Name of the person to greet.
-      required: false
```

Comment:
All of the other changes make sense to me (adding descriptions, clarifying style and explode, etc.
But this is a change in the actual content and intent of the endpoint. Can you say more about why you did this?

Reply:
Per the OpenAPI specification, path parameters _must_ have `required: true`. The nominal purpose of this test is to "check if add_operation is called only on http methods field", which does not obviously require that the name parameter by optional, so it seemed reasonable to make the name required. But perhaps there is some deeper purpose that I'm missing.

Reply:
ah yep - I missed that it was a path param. :+1: